params/config.go

```diff
@@ -292,20 +295,31 @@ type NodeConfig struct {
 	// discoverable peers with the discovery limits.
 	RequireTopics map[discv5.Topic]Limits `json:"RequireTopics"`
 
-	// StatusServiceEnabled enables status service api
-	StatusServiceEnabled bool
-
-	// InstallationId id of the current installation
-	InstallationID string
-
-	// DebugAPIEnabled enables debug api
-	DebugAPIEnabled bool
-
 	// MailServerRegistryAddress is the MailServerRegistry contract address
 	MailServerRegistryAddress string
+}
 
+// ShhextConfig defines options used by shhext service.
+type ShhextConfig struct {
+	// BackupDisabledDataDir is the file system folder the node should use for any data storage needs that it doesn't want backed up.
+	BackupDisabledDataDir string `validate:"required"`
```

Comment:
as in iOS back up? so it isn't in the Documents folder?
what is the difference between this and some temporary storage?

is it for temp files or for very sensitive data?

Reply:
i guess this directory is for sensitive data. it is used for PFS service

Reply:
so maybe we can call it something like `SensitiveDataDir` or something similar?

Reply:
sounds ok to me. i can rename. @cammellos what do you think?

Reply:
I am ok with renaming it, to me personally is clearer to call it `backupdisableddatadir`, as that's what we are interested in (confidential information might also be stored in `~/Documents`, although they will be backed up), on android is literally called `no_backup`, but don't feel strongly about it to be honest